explainers/prioritized-task-scheduling.md

```diff
@@ -0,0 +1,883 @@
+  const newSignal = TaskSignal.any([signal], {priority: 'background'});
+  scheduler.postTask(otherTask, {signal: newSignal});
+}
+```
+
+**Note**: the `signal` option for `scheduler.postTask()` and other `scheduler` APIs is specified as
+an `AbortSignal`, which means either a plain `AbortSignal` or a `TaskSignal` can be used, since
+`TaskSignal` inherits from `AbortSignal`. This allows these APIs to work with existing code that
+uses `AbortSignal`. It also means a `TaskSignal` can be passed as the `signal` option to other APIs
+that take an `AbortSignal`.
+
+```js
+function doSomethingWithAbortSignal(signal) {
+  // This will work if signal is either an AbortSignal or TaskSignal.
+  scheduler.postTask(task, {signal});
+}
+
+// ... somewhere else ...
+
+const controller = new AbortController();
+const signal = controller.signal;
+fetch(someUrl, {signal});
+doSomethingWithAbortSignal(signal);
+```
+
+### `scheduler.yield()`
+
+(See also the [separate `scheduler.yield()` explainer](./yield-and-continuation.md)).
+
+`scheduler.yield()` can be used in any context to yield to the event loop by awaiting the promise it
+returns. The [task continuation](#definitions-and-concepts) &mdash; the code that runs as a
+microtask when the returned promise is resolved &mdash; runs in a new browser task and gives the
+browser a scheduling opportunity. Continuations are [given a higher
+priority](#how-continuations-are-prioritized) by the UA, which helps minimize the latency penalty
+for yielding.
+
+Whereas `scheduler.postTask()` can be used to break up long tasks on function boundaries by
+scheduling chunks of work, `scheduler.yield()` can be used to break up long tasks by inserting
+yield points in functions.
+
+**Example: Inserting yield points.**
+```js
+async function task() {
+  doWork();
+  // Yield to the event loop and resume in a new browser task.
+  await scheduler.yield();
+  doMoreWork();
+
+  await scheduler.yield();
+  // ... and so on ...
+}
+
+// Schedule the long but yieldy task to run. scheduler.yield() can be used to
+// break up long timers, long I/O callbacks, etc.
+setTimeout(task, 100);
+```
+
+<hr/>
+
+Similar to `scheduler.postTask()`, developers can provide `{signal, priority}` options to control
+continuation scheduling and cancellation.
+
+**Example: Controlling continuations with priority and signal.**
+
+```js
+const controller = new TaskController({priority: 'background'});
+
+async function task() {
+  doWork();
+
+  // Deprioritize the continuation, and reject the promise if
+  // the signal is aborted.
+  await scheduler.yield({signal: controller.signal}});
+
+  doMoreWork();
+
+  // Deprioritize the continuation, but don't ever abort it.
+  await scheduler.yield({priority: 'background'}});
+
+  ...
+}
+```
+
+<hr/>
+
+If the yielding task was originally scheduled with `scheduler.postTask()` and no options are passed
+to `scheduler.yield()`, then the current priority/signal will be **inherited**. This works across
```

Comment:
Does inheritance only work for `yield()`? Or does it also work for `postTask()`?